atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -45,15 +45,29 @@
 from atomic_reactor import start_time as atomic_reactor_start_time
 from atomic_reactor.plugin import PreBuildPlugin
 from atomic_reactor.constants import INSPECT_CONFIG
-from atomic_reactor.util import get_docker_architecture
+from atomic_reactor.util import get_docker_architecture, _PREFERRED_LABELS
 import json
 import datetime
 
 
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
 
-    def __init__(self, tasker, workflow, labels, 
+    DEFAULT_ALIASES = {
+        'Name': 'name',
+        'Version': 'version',
+        'Release': 'release',
+        'Architecture': 'architecture',
+        'Vendor': 'vendor',
+        'RUN': 'run',
+        'INSTALL': 'install',
+        'UNINSTALL': 'uninstall',
```

Comment:
OK, these atomic CLI labels are accepted as upper-case but preferred as lower-case, so this what you have here is correct.
https://github.com/projectatomic/ContainerApplicationGenericLabels/issues/8